src/tooltip.jsx

```diff
@@ -26,6 +26,10 @@ let Tooltip = React.createClass({
     this._setTooltipPosition();
   },
 
+  componentWillReceiveProps() {
```

Comment:
Thanks @CumpsD - I think I might have misspoken on gitter because this may have to be done in `componentDidUpdate`. Is this working for you?